src/utils.rs

```diff
@@ -70,14 +70,14 @@ pub fn calculate_relocated_name(mut close_nodes: Vec<XorName>, original_name: &X
 
 #[cfg(test)]
 mod tests {
-    use peer_manager::MIN_GROUP_SIZE;
     use rand;
     use routing_table::Xorable;
     use rust_sodium::crypto::hash::sha256;
     use xor_name::XorName;
 
     #[test]
     fn calculate_relocated_name() {
+        let min_group_size = 8;
```

Comment:
I feel like it could be a constant per file instead of per test in many cases. But we can decide about those individually later, too.